devops/scripts/deploy_shared_service.sh

```diff
@@ -67,6 +72,11 @@ if [ -z "${access_token:-}" ]; then
   access_token=${ACCESS_TOKEN}
 fi
 
+if [ -n "${properties:-}" ]; then
+  # If properties are passed, prepend with a comma so we can add to the payload proeprties
+  properties=", $properties"
```

Comment:
prepending with a comma also feels a little brittle

Reply:
I'm open to suggestions...?